modules/news.ts

```diff
@@ -0,0 +1,139 @@
+import STRINGS from "../lib/db.js";
+import Client from "../sidekick/client";
+import { proto } from "@adiwajshing/baileys";
+import BotsApp from "../sidekick/sidekick";
+import { MessageType } from "../sidekick/message-type";
+import inputSanitization from "../sidekick/input-sanitization";
+import Axios from "axios";
+import config from "../config";
+const NEWS = STRINGS.news;
+
+module.exports = {
+    name: "news",
+    description: NEWS.DESCRIPTION,
+    extendedDescription: NEWS.EXTENDED_DESCRIPTION,
+    demo: { isEnabled: false },
+    async handle(
+        client: Client,
+        chat: proto.IWebMessageInfo,
+        BotsApp: BotsApp,
+        args: string[]
+    ): Promise<void> {
+        /*******************************************
+         *
+         *
+         * Functions
+         *
+         *
+         ********************************************/
+        const checkPub = async (newsList: string[], requestedPub: string) => {
+            for (let i = 0; i < newsList.length; i++) {
+                if (newsList[i].toUpperCase == requestedPub.toUpperCase) {
+                    return newsList[i];
+                }
+                return;
+            }
+        };
+        /*******************************************
+         *
+         *
+         * Actions
+         *
+         *
+         ********************************************/
+        if (args.length === 0) {
+            await client
+                .sendMessage(BotsApp.chatId, NEWS.NO_COMMMAND, MessageType.text)
+                .catch((err) => inputSanitization.handleError(err, client, BotsApp));
+            return;
+        }
+        if (args[0] == "help") {
+            await client
+                .sendMessage(
+                    BotsApp.chatId,
+                    NEWS.EXTENDED_DESCRIPTION,
+                    MessageType.text
+                )
+                .catch((err) => inputSanitization.handleError(err, client, BotsApp));
+            return;
+        }
+        if (args[0] == "search") {
+            args.shift();
+            var searchTerm = args.join(" ");
+            let searchResponse;
+            try {
+                await Axios.get(
+                    config.NEWS_API_URL + "news-list?searchTerm=" + searchTerm
+                )
+                    .then((res) => {
+                        searchResponse = res.data;
+                    })
+                    .catch((error) => {
+                        console.error(error);
+                    });
+            } catch (error) {
+                throw error;
+            }
+
+            for (let i = 0; i < searchResponse.length; i++) {
+                searchResponse[i] = `${i + 1}` + ".)  " + `${searchResponse[i]}`;
+            }
+
+            let message = searchResponse.join("\n\n");
```

Comment:
```suggestion
            let message = "```The following publications are available with the term```" + "*searchTerm* ```in it:```" + searchResponse.join("\n\n");
```

This would be neater, do add this part in a neater manner.